model/pci_device.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class PciDevice < Sequel::Model
```

Comment:
The new model and migration should be in separate commits for zero downtime deployment. Otherwise, the app won't start in production until you run the migration. We initially froze all models.

```
➜  ubicloud git:(ben/learn_gpu) RACK_ENV=production bundle exec rackup
bundler: failed to load command: rackup (/Users/enescakir/.asdf/installs/ruby/3.2.3/bin/rackup)
/Users/enescakir/.asdf/installs/ruby/3.2.3/lib/ruby/gems/3.2.0/gems/sequel-5.79.0/lib/sequel/plugins/require_valid_schema.rb:58:in `get_db_schema_array': Not able to parse schema for model: PciDevice, table: "pci_device" (Sequel::Error)
        from /Users/enescakir/.asdf/installs/ruby/3.2.3/lib/ruby/gems/3.2.0/gems/sequel-5.79.0/lib/sequel/model/base.rb:792:in `get_db_schema'
        from /Users/enescakir/.asdf/installs/ruby/3.2.3/lib/ruby/gems/3.2.0/gems/sequel-5.79.0/lib/sequel/model/base.rb:606:in `set_dataset'
        from /Users/enescakir/.asdf/installs/ruby/3.2.3/lib/ruby/gems/3.2.0/gems/sequel-5.79.0/lib/sequel/model/plugins.rb:52:in `block in after_set_data
```

Reply:
thank you!